automation/taskcluster/lib/tasks.py

```diff
@@ -593,12 +591,18 @@ def _craft_raptor_task(
                     "MOZILLA_BUILD_URL": apk_location,
                     "NEED_XVFB": "false",
                     "NO_FAIL_ON_TEST_ERRORS": "1",
-                    "TASKCLUSTER_WORKER_TYPE": 'proj-autophone/{}'.format(worker_type),
-                    "WORKING_DIR": "/builds/worker",
-                    "WORKSPACE": "/builds/worker/workspace",
+                    "SCCACHE_DISABLE": "1",
+                    "TASKCLUSTER_WORKER_TYPE": worker_type[len('gecko-'):],
+                    "TRY_COMMIT_MSG": "",
```

Comment:
Nit: Remove the `TRY_*` arguments. I don't think we need them for regular puses.